docs/source/tracking.rst

```diff
@@ -156,6 +158,21 @@ environment variable.
         mlflow.log_param("a", 1)
```

Comment:
documentation for `log_param` in `fluent` says that value should be a `string` (or maybe we should document that param value gets cast to string in service)